### ISSUE TITLE
Add a toggle to expand the side 

### ISSUE THREAD

**Issue by ghost** [searxng/searxng#2598]
**Is your feature request related to a problem? Please describe.**

I'm always frustrated when on each search that I make, I have to click on "Suggestions" to see better suggestions to find the result that I need, or I have to click on "Messages from the search engines" to see what engines are not working or which engine is giving me an error.
**Describe the solution you'd like**

Add a toggle in the setting to expand "Suggestions" for example, so the user can choose what they want to see every time they make a search query, or maybe someone doesn't want "info" to be shown by default, so they can easily turn it off.

**Describe alternatives you've considered**

**Additional context**

![Screenshot 2023-08-05 142710](https://github.com/searxng/searxng/assets/85373847/2df2a6e6-e705-45bd-988c-8e940d26c045)

**Comment by HexagonWin** [comment]
I'm also using the simple theme now and this is really frustrating. Is there any hidden option to enable this feature?

**Comment by unixfox** [comment]
I'm in a favor of having suggestions expanded by default. I don't see why they are even hidden when the suggestions on google for example is displayed to all the users.

**Comment by return42** [comment]
> I don't see why they are even hidden 

By folding out the sidebar elements, the UI looks less cluttered.  Especially on small devices like smartphones, where the sidebar is above the results list, the UX should be improved [#2140].

- https://github.com/searxng/searxng/issues/2140
- https://github.com/searxng/searxng/pull/2370

**Comment by unixfox** [comment]
> By folding out the sidebar elements, the UI looks less cluttered. Especially on small devices like smartphones, where the sidebar is above the results list, the UX should be improved [#2140].

We could expand only the tabs on desktop and not mobile devices.

**Comment by return42** [comment]
> We could expand only the tabs on desktop and not mobile devices.

Yes, but not sure if it will solve the request from the opener who sends a screenshot from a mobile device.  It might be better to have a client solution that stores if a item was collapsed or opened.

It might be useful if we had a way (at least in the JS clients) to save settings so that the user does not have to change them again with every request.

**Comment by inson1** [comment]
Why not both?

**Comment by return42** [comment]
> Why not both?

Today we have (what feels like) hundreds of switches that are only used in niches, but hinder further development.
